common/system/tasks/buzzer_heartbeat.cpp

```diff
@@ -1,25 +1,59 @@
-
 #include "buzzer_heartbeat.h"
 
-#include "hardware_manager.h"
-
 static uint32_t last_time = 0;
-static bool isOn;
+// static bool isOn;
 static int buzzerId_ = 0;
+BuzzerState state = BuzzerState::PYRO;
```

Comment:
`buzzerHeartbeat_tick` gets the state each time so I don't think it needs to be stored here